ansible_rulebook/json_generator.py

```diff
@@ -161,6 +161,22 @@ def visit_condition(parsed_condition: ConditionTypes, variables: Dict):
                     ),
                 }
             }
+        elif parsed_condition.operator == "in":
+            return create_binary_node(
```

Comment:
can we refactor all existing applicable operators to use the same create_binary_node method?

Reply:
@bzwei In a separate PR since its unrelated to these changes.

Reply:
@bzwei https://github.com/ansible/ansible-rulebook/pull/267